lib/redis/namespace.rb

```diff
@@ -364,6 +364,15 @@ def call_with_namespace(command, *args, &block)
 
       (before, after) = handling
 
+      # Avoid modifying the caller's arguments.
+      args = args.map do |arg|
```

Comment:
modifying `*args` parameters doesn't modify the caller's arguments, so the copy in `map` is unnecessary